assignment6/index.php

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+  <title>Assignment 6</title>
+  <meta charset="utf-8"> 
+</head>
+<body>
+	   <!--Dissa Kondo
+     index.html
+     INFO1311.WW
+     Lisa THOENDEL 
+     12/14/19
+-->
+<h1>Hello, world!</h1>
+<div id="result"></div>
+<div>Allow me to introduce myself! My name is <?php echo "Dissa Kondo!"; ?></div>
```

Comment:
good php statements! complete with quotation marks and that necessary semicolon! 👍